### PR TITLE
Merge dev into main

### PR DESCRIPTION
**Pull request by Pronyo-Chan**

### CODE REVIEW

File: rando_modules/logic.py [hunk 1]
```diff
@@ -552,38 +566,39 @@ def _generate_item_pools(
             Item.get(Item.item_name == "PouchE"),
         ]
 
-        cnt_items_removed = 0
+        pool_other_items.extend(pouch_items)
+
+    # Add unused badge duplicates, if needed
+    if add_unused_badge_duplicates:
+        unused_badge_duplicates = []
+        for item in Item.select().where(Item.unused_duplicates == True):
```
Comment (Pronyo-Chan) [inline rando_modules/logic.py:574]: Not sure of the actual syntax, but surely there's a way to select the items and directly assign them to unused_badge_duplicates without iterating in a for loop?

**Review by Pronyo-Chan — COMMENTED** [review]
lgtm

File: rando_modules/logic.py (hunk 1 above)
Comment (icebound777) [reply rando_modules/logic.py:574]: Annoyingly, this is how peewee handles Selects in their ORM, you only get the data you want by iterating over it manually.
For reference see http://docs.peewee-orm.com/en/latest/peewee/querying.html#selecting-multiple-records

File: metadata/item_general.py [hunk 2]
```diff
@@ -373,13 +390,13 @@
     "Quick Change",
     "D Down Pound",
     "Dizzy Stomp",
-    "Smash Charge",
```
Comment (HailSanta) [inline metadata/item_general.py:376]: Is this change intentional? (same with Jump Charge a few lines down)

Comment (icebound777) [reply metadata/item_general.py:376]: It is. The items those two lines represent are actually the beta charge badges, but were misnamed before. The regular jump and smash charge show up elsewhere on the same list.
This list reproduces the data array in the mod repo's MainMenu.patch file, which is part of the seed hash icons on the save select screen.

**Comment by icebound777** [comment]
lgtm
